addons/point_of_sale/static/src/js/models.js

```diff
@@ -1536,8 +1536,8 @@ exports.Product = Backbone.Model.extend({
             return (! item.product_tmpl_id || item.product_tmpl_id[0] === self.product_tmpl_id) &&
                    (! item.product_id || item.product_id[0] === self.id) &&
                    (! item.categ_id || _.contains(category_ids, item.categ_id[0])) &&
-                   (! item.date_start || moment(item.date_start).isSameOrBefore(date)) &&
-                   (! item.date_end || moment(item.date_end).isSameOrAfter(date));
+                   (! item.date_start || moment.utc(item.date_start).isSameOrBefore(date)) &&
```

Comment:
date_start and date_end are a "Date" in 13.0, not a DateTime, so the utc stuff is probably wrong as date are not supposed to be subject to timezone. Maybe we should find a way to only have the date and not a datetime.

BTW, after 13.0, it is a datetime so no issues and no need of the uct() call.